File: Solution_Practice_I/java/prob4_9.java
import java.util.Scanner;
public class prob4_9  {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int num, dig, aux=0, neww=1,cont=0;
        do {
            System.out.println("Ingrese cuantos numeros ver:");
            num=input.nextInt();
        } while (num<0);
        while (num>0){
            aux+=1;
            cont=0;
            for (int i=1; i<=aux; i++){
                if(aux%i==0) cont+=1;
            }
            if (cont==2){
                dig=aux%10;
                System.out.print(neww+" ");
                neww+=dig;
                num-=1;
            }
        }

        input.close();
    }
}
//Notinsky Javierinsky: empieza en 1 y se suma el ultimo digito de los numeros primos

File: Solution_Practice_I/python/prob4_9.py
while True:
    num=int(input("Ingrese cuantos numeros ver: "))
    if num>=0:
        break
aux=0
new=1
while num>0:
    aux+=1
    cont=0
    for i in range(1, aux+1):
        if aux%i==0:cont+=1
    if cont==2:
        dig=aux%10
        print(new, end=" ")
        new+=dig
        num-=1

#Notinsky Javierinsky: empieza en 1 y se suma el ultimo digito de los numeros primos